package.json

```
@@ -67,7 +67,7 @@
     "@synonymdev/slashtags-widget-facts-feed": "1.1.0",
     "@synonymdev/slashtags-widget-news-feed": "1.1.0",
     "@synonymdev/slashtags-widget-price-feed": "1.1.0",
-    "@synonymdev/web-relay": "1.0.0",
+    "@synonymdev/web-relay": "^1.0.2",
```

```suggestion
    "@synonymdev/web-relay": "1.0.2",
```
Let's remove the ^ to ensure we have explicit control over dependency versions.